pages/portfolio.html

```diff
@@ -1,12 +1,177 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+        <title>Jaime Mitchell</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" type="text/css" href="../styles/portfolio.css" />
+        <title>Portfolio/Resume</title>
 </head>
-<body>
-  
+
+<body class="container">
+        <!--HEADER-->
+        <header>
+                <h1>Jaime Mitchell</h1>
+                <h2>Software Engineer</h2>
+                <a href="https://www.linkedin.com/in/jaimemitchell1/" target="_blank">
+                        <h3>LinkedIn</h3>
+                </a>
+        </header>
+
+        <nav>
+                <li><a href="../index.html">HOME</a></li>
+                <li><a href="./about.html">ABOUT</a></li>
+                <li><a href="./portfolio.html">PORTFOLIO</a></li>
+        </nav>
+
+        <section class="main">
+                <!--RELEVANT HARD SKILLS-->
+                <ul>
+                        <h3>HARD SKILLS</h3>
+                        <h4>April 2022 - Present</h4>
```

Comment:
The only common tags allowed under a `ul` are `li` tags. These `h3` and `h4` tags here and in later sections cause a validation error. When I moved them out of (above) the `ul`, the spacing looked quite different, which is probably why you put them where you did, but even if a browser seems to be able to display the markup, a different browser may show it differently (since the display of an invalid html page is technically undefined).

So I'd pull these out of the `ul`, and then try to get the layout back to how you want it using styling.